addons/web/static/src/core/notifications/notification.scss

```diff
@@ -3,16 +3,16 @@
 // The notification just add overlays.
 
 .o_notification_manager {
-    @include o-position-absolute($o-navbar-height * 1.15, map-get($spacers, 2), $left: calc(100vw - #{$o-notification-max-width}));
+    @include o-position-absolute($top: $o-navbar-height * 1.15, $right: $o-notification-margin, $left: $o-notification-margin);
     z-index: $o-notification-zindex;
 
-    .o_notification {
-        border-left-width: $o-notification-border-left-width !important; // Overrides border-[x] utility classes
-        box-shadow: $o-notification-shadow;
+    @include media-breakpoint-up(sm) {
+        left: auto;
+        width: $o-notification-max-width-md;
     }
 
-    .o_notification_close {
-        @include o-position-absolute(0, 0);
+    .o_notification_border {
```

Comment:
I'm wondering if we should use another name for this element and its variable... each time I check this PR I'm mislead to think that's about the actual card border.

 `o_notification_flag`?  `o_notification_bar`? What do you think?

Reply:
notification bar seems better indeed 👍 
Thanks !